ocean/transects/compute_transects.py

```diff
@@ -21,6 +20,8 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import xarray as xr
```

Comment:
Should we just completely remove `xr` from this script for now?

I'm guessing you went with `netCDF4` over `xr` because it can open files more quickly, correct?